components/dashboards/org.wso2.carbon.dashboards.core/pom.xml

```diff
@@ -126,7 +136,9 @@
             org.osgi.service.component.annotations.*; version="${org.osgi.service.component.annotations.version.range}",
             org.slf4j.*; version="${slf4j.version.range}",
             com.google.gson.*; version="${gson.version.range}",
-            org.yaml.snakeyaml.*; version="${org.yaml.version.range}"
+            org.yaml.snakeyaml.*; version="${org.yaml.version.range}",
+            org.wso2.carbon.analytics.permissions.*,
```

Comment:
Please add `version` for these OSGi imports.

Reply:
Ack